docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Using Machine ID With Github Actions
+description: A tutorial for using Machinie ID with Github Actions
+---
+
+Github actions are a popular CI/CD platform that works as a larger part of the Github
```

Comment:
I think we should use "GitHub Actions" uniformly across the guide to keep GitHub happy.